.gitignore

```diff
@@ -41,6 +41,7 @@ ormlogs.log
 
 # Using either dev or prod docker-compose file
 docker-compose.yml
+!/e2e/setupIpfsCluster/docker-compose.yml
```

Comment:
Minor point, but wouldn't it make sense to have this below the other line that was added? Why is this in a different section of the file?